src/Form/TextStyleForm.php

```diff
@@ -235,46 +235,44 @@ public function form(array $form, FormStateInterface $form_state) {
         'justified' => 'Justified',
       ],
       '#default_value' => '',
-      '#description' => $this->t('')
+      '#description' => $this->t(''),
     ];
     $form['lineHeight'] = [
       '#type' => 'textfield',
       '#title' => $this->t(''),
       '#default_value' => '',
-      '#description' => $this->t('')
+      '#description' => $this->t(''),
```

Comment:
Unrelated: so many fields with empty titles and descriptions. Is this intended?

Reply:
Right, we need them for component styling. we don't use them yet. but I can add a title for now.